components/06-examples/aanvragen.hbs

```diff
@@ -1,13 +1,14 @@
-{{render '@header' }}
+{{> '@header' }}
 
 <main>
   <div class="row">
     <div class="col-xs-12">
-      <h1>Project | Aanbouw Kerkstraat 1</h1>
+      <h1>Aanvragen</h1>
+        {{render '@steps'}}
```

Comment:
De indenting klopt hier niet: twee spaces teveel

Reply:
Deze is uitgevoerd. Zo niet corrigeer me waar nodig.